marathon/marathon.go

```diff
@@ -212,12 +212,27 @@ func (m Marathon) url(path string) string {
 
 type params map[string][]string
 
+// urlWithQuery was altered to handle a case when marathon location has a path in addition
```

Comment:
Nice comment. I'd prefer to change it a little. Comment should describes what and why function is doing not what happen to the function. Try to fill following template:

```
urlWithQuery returns ...
If location is given with path e.g, .... then ...
```